File: Python/6. Itertools/exercise1.py
# itertools.product() "https://www.hackerrank.com/challenges/itertools-product/problem"

# Enter your code here. Read input from STDIN. Print output to STDOUT
from itertools import product

a = list(map(int, input().split()))
b = list(map(int, input().split()))

print(*product(a, b))

File: C/3. Arrays and Strings/exercise1.c
// 1D Arrays in C "https://www.hackerrank.com/challenges/1d-arrays-in-c/problem"

#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    int n;
    int sum = 0;

    scanf("%d", &n);
    int *arr = (int*)malloc(n * sizeof(int));

    for (int i=0; i<n; i++) {
        scanf("%d", &arr[i]);
        sum += arr[i];
    }
    printf("%d", sum);
    free(arr);
    return 0;
}
